scss/standalone/patterns_segmented-control.scss

```diff
@@ -1,7 +1,7 @@
 @import '../vanilla';
 @include vf-base;
 
-@include vf-p-tab-buttons;
+@include vf-p-segmented-control;
```

Comment:
I'm not sure if we handled it when renaming "status label" but we should be careful when removing/renaming mixins. Some project may use them.

Ideally `vf-p-tab-buttons` should still be in Vanilla to avoid breaking someone's build. (But I can also try to search if anyone is actually using it outside of Vanilla).

Reply:
The `vf-p-tab-buttons` doesn't seem to be used elsewhere, but to be on a safe side I'd suggest keeping it with something like:

```
@mixin vf-p-tab-buttons {
   @include vf-p-segmented-control;
}
```

Reply:
@bartaz Do you think it'd be better to add this in a separate file, or in the same one?